Source/SelectionsGallery/YPSelectionsGalleryCell.swift

```diff
@@ -54,6 +54,20 @@ public class YPSelectionsGalleryCell: UICollectionViewCell {
         self.editSquare.isHidden = !editable
     }
     
+    func addRemoveButton(target: Any?, action: Selector) {
+        let image = YPConfig.icons.removeImage
+        let removeButton = UIButton(frame: CGRect(origin: CGPoint.zero, size: image.size))
+        removeButton.setBackgroundImage(image, for: UIControl.State())
+        removeButton.translatesAutoresizingMaskIntoConstraints = false
+        
+        self.addSubview(removeButton)
+        ["H:[subview]-12-|", "V:|-12-[subview]"].forEach { visualFormat in
```

Comment:
Please change to https://github.com/freshOS/Stevia/issues for constraints here.

Reply:
I'll take care of the changes :)